src/configure.py

```diff
@@ -504,10 +504,16 @@ def main(args):
 
 if __name__ == "__main__":
 
-    if not sys.version_info.major >= 3 and sys.version_info.minor >= 6:
+    if (
+        not sys.version_info.major >= PYTHON_MAJOR
+        and sys.version_info.minor >= PYTHON_MINOR
+    ):
```

Comment:
This was marked as resolved even though nothing was changed. But the same code found in `src/main.py` (lines 117-120) was changed as per my suggestion. Perhaps I am missing some nuance to the difference between this code and the code in `src/main.py`, but I just wanted to flag this for a double check that this code is written as intended, because it still looks wrong to me.

Reply:
Fixed now, thank you.